tajo-core/src/test/java/org/apache/tajo/QueryTestCaseBase.java

```diff
@@ -439,7 +437,18 @@ public ResultSet executeJsonFile(String jsonFileName) throws Exception {
    * @param result Query result to be compared.
    */
   public final void assertResultSet(ResultSet result) throws IOException {
-    assertResultSet("Result Verification", result, getMethodName() + ".result");
+    assertResultSet(result, false);
+  }
+
+  /**
+   * Assert the equivalence between the expected result and an actual query result.
+   * If it isn't it throws an AssertionError.
+   *
+   * @param result Query result to be compared.
+   * @param ignoreOrdering If true, don't concern ordering.
+   */
+  public final void assertResultSet(ResultSet result, boolean ignoreOrdering) throws IOException {
```

Comment:
I think that this is not a good convention. We need to make tajo to be more determinant including tests. 
How about adding order-by clauses to test queries?